modules/installing-ocp-agent-tui.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-tui_{context}"]
+= Using the `agent-tui` application to check for network configuration errors
+
+After you boot the agent image and network services are made available to the host, the `agent-tui` application performs checks to verify that the host can retrieve release images.
+
+If there are internal network configuration issues that may cause an installation to fail, you can use the `agent-tui` to make adjustments to your network configurations.
+
+.Procedure
+
+. Wait for the `agent-tui` to check for network configuration errors.
++
+[NOTE]
+====
+The `agent-tui` is intended to check for network configuration errors that prevent the deployment host from retrieving release images. It cannot detect every type of issue that will cause an installation to fail, such as configuring an incorrect rendezvous host IP address.
+====
+
+. If the `agent-tui` states that "Agent-based installer connectivity checks have passed", either wait for the prompt to time out or select *No*. The `agent-tui` application closes and the installation proceeds.
++
+[NOTE]
+====
+You may still choose to view or change network configuration settings even if the connectivity checks have passed.
+
+However, if you choose to interact with the `agent-tui` rather than letting it time out, you must manually quit the TUI to proceed with the installation.
+====
+
+. If the `agent-tui` checks have failed, use the following steps to reconfigure the host's network settings:
+
+.. Read the `Check Errors` section of the TUI.
+This section displays error messages specific to the failed checks.
```

Comment:
Again, a sample picture here may help

Reply:
Do you think you could provide me a good quality screenshot of the console's home screen when it displays failed checks and error messages?